test_conformance/basic/test_explicit_s2v.cpp

```diff
@@ -13,6 +13,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+#include <cmath>
+#define isnan std::isnan
```

Comment:
I'd much rather just spell out `std::isnan` in the places where it is used, but if we're going to keep something like this, I'd suggest using a more C++-like solution, something like:

```c++
using std::isnan;
```